### ISSUE TITLE
Bioconda packaging

### ISSUE THREAD

**Issue by Emiller88** [hyulab/PINTS#3]
Awesome work! I'd love to package this up in bioconda for use in https://github.com/nf-core/nascent

```shell
conda skeleton pypi pypints
```
fails with
```
Error: No source urls found for pypints
```

Could you add a source url to pypi?

**Comment by liyao001** [comment]
Hi Emiller88, thank you for helping port PINTS into bioconda and ease the burden on data analysis. I can reproduce the error posted above; I'll look into this issue and let you know if I have progress.

**Comment by liyao001** [comment]
Hi Emiller88, we solved the problem and here is the story: previously, we only uploaded the package in wheel (`bdist`) format but not the source tarball (`sdist`) to PyPI; even though the contents in the two files are almost identical for PINTS, conda-build needs the source tarball file. We have [updated our CI/CD pipeline](https://github.com/hyulab/PINTS/commit/3ca4626b76447c978e84b4aaec65ad3cbfd324b3), and from now on, when we update the source code, both wheel and source tarball will be uploaded to PyPI. We tested the fix by generating a recipe with `conda skeleton` and pushing the commit to bioconda's repo. Hopefully, PINTS will be available on bioconda soon once the bioconda community accepts the [pull request](https://github.com/bioconda/bioconda-recipes/pull/33742).
